{"name": "Успешное изменение name авторизованного пользователя", "status": "passed", "description": "При попытке сменить name у авторизованного пользователя, изменение данных происходит успешно", "start": 1739735812192, "stop": 1739735812272, "uuid": "914bf015-c87f-4cb1-814a-0e1baf0a22ff", "historyId": "244c6130c4bc20f55b5e843190ecbe4c", "testCaseId": "244c6130c4bc20f55b5e843190ecbe4c", "fullName": "tests.test_changing_user_data.TestChangingUserData#test_changing_user_name_with_auth", "labels": [{"name": "suite", "value": "Изменение данных пользовователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestChangingUserData"}, {"name": "host", "value": "nb-122695"}, {"name": "thread", "value": "20624-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_changing_user_data"}]}